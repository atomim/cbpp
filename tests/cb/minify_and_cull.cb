// dis a function
function getTrackFilename(id)
	return "sync\synctrack" + id + ".dat"
endFunction
