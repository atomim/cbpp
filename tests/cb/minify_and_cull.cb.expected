function getTrackFilename(id)
	return "sync\synctrack"+id+".dat"
endFunction